examples/web-app/src/components/Home/Home.js

```diff
@@ -0,0 +1,196 @@
+import React, { useState, useEffect } from "react";
+import { UserFactory } from "nightfall-sdk";
+import "./Home.css";
+
+export default function User() {
+  const [userAddress, setUserAddress] = useState();
+  const [userNightfallAddress, setUserNightfallAddress] = useState();
+  const [userRecipient, setUserRecipient] = useState();
+  const [nightfallMnemonic, setNightfallMnemonic] = useState();
+  const [clientApiUrl, setClientApiUrl] = useState();
+  const [nightfallBalances, setNightfallBalances] = useState(0);
+  const [nightfallMnemonicRecepient, setNightfallMnemonicRecepient] =
+    useState(null);
+
+  const tokenContractAddress = "0xa8473bEF03cBE50229a39718CBDC1fdee2F26b1a";
+  const tokenErcStandard = "ERC20";
+  const txValue = "0.0001";
+
+  let createUserFirstTime = async () => {
+    const nightfallUser = await UserFactory.create({
+      clientApiUrl: "http://localhost:8080",
```

Comment:
We probably need some kind of env detection. Do we have an intention to ever use this app outside of local env?

Reply:
I think that it would be nice to have the app deployed and live for showcasing some of the features. That is why i am not using .env file. This way it also makes sense to drop the example-scripts and the web-app from the next release of the NPM packaged, as we discussed with @luizoamorim . Would like to hear @imagobea opinion on this.